pom.xml

```diff
@@ -66,6 +68,14 @@
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.codehaus.mojo</groupId>
+        <artifactId>findbugs-maven-plugin</artifactId>
+        <configuration>
```

Comment:
File leaks are treated as low severity by FindBugs? Maybe that is what should be fixed.